.github/workflows/publish-staging-redacted-export.yml

```
@@ -0,0 +1,140 @@
+name: Publish Staging redacted export to S3
+
+on:
+  # workflow_dispatch:
+  # TODO: Switch to manual dispatch
+  pull_request:
+
+jobs:
+  redact_and_upload_export:
+    name: Redact and upload database export
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.17.x
```

Due to https://github.com/UKGovernmentBEIS/beis-opss-psd/pull/1612
```suggestion
          node-version: 14.18.x
```